images/fedora/f30/missing-docs

```diff
@@ -0,0 +1,18 @@
+acl
+bash
+chkconfig
+curl
+dbus-daemon
```

Comment:
Umm... I wonder how I have `/usr/share/man/man8/dnf.8.gz` in my F30 toolbox that was created using the `fedora-toolbox:30` image that's already on `registry.fedoraproject.org` [1]. On the other hand, if I use an image built locally using `buildah bud ...`, I don't have the file.

Looking at a `fedora:30` image [1], I don't see a `/usr/share/man/man8/dnf.8.gz` either.

Any idea?

[1]  `$ toolbox create --release 30`
[2]  `$ podman run -it --rm registry.fedoraproject.org/fedora:30 /bin/bash`

Reply:
I have a feeling that there's something weird about the `fedora-toolbox:30` image on the registry. There's no `dbus-daemon` package in the `fedora:30` image, but it's there in the toolbox image.

However, we can't push a new image either because currently the F30 and F31 container builds are in a strangely broken limbo. Such is life, I guess. :)